src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsReconciliationService.kt

```diff
@@ -96,33 +96,35 @@ class AdjudicationsReconciliationService(
   private suspend fun mismatchNotDueToAMerge(prisonerId: ActivePrisonerId, nomisSummary: AdjudicationADAAwardSummaryResponse, dpsAdjudications: List<ReportedAdjudicationDto>): Boolean {
     val merges = nomisApiService.mergesSinceDate(prisonerId.offenderNo, nomisMigrationDate)
     if (merges.isNotEmpty()) {
-      log.debug("Merges found for {} the latest on {}", prisonerId.offenderNo, merges.last().requestDateTime)
       val adjudicationsNotPresentOnDPSBooking = adjudicationsNotPresentOnDPSBooking(nomisSummary, dpsAdjudications)
-      log.debug("Missing adjudications on DPS booking {}", adjudicationsNotPresentOnDPSBooking)
       if (adjudicationsNotPresentOnDPSBooking.isNotEmpty()) {
+        val dpsAdjudicationsOnPreviousBooking = merges.flatMap { doApiCallWithRetries { adjudicationsApiService.getAdjudicationsByBookingId(it.previousBookingId) } }
+        val allAdjudications = dpsAdjudicationsOnPreviousBooking + dpsAdjudications
         val nomisAdaSummary = nomisSummary.toAdaSummary()
```

Comment:
the meat of the change: add in all the adjudications fro, all the previous bookings that were merged (there is typically just one previous booking but could be more for mutiple merges).
NOMIS would have moved the adjudication to the current booking, DPS would have kept them on the old booking.